### DISCUSSION TITLE
portable transmission

### DISCUSSION THREAD

**Discussion by eooeo** [transmission/transmission#2951] [Q&A]
Hello, help me please. Is It possible to make the transmission portable? Thank you.

**Comment by vchimishuk** [comment]
@eooeo, what do you mean "portable"? Would you like to provide some details, please?

**Comment by eooeo** [comment]
**wiki** 
_A portable application is software that does not require an installation procedure to run and can be completely stored on removable media, which allows you to use this software on many computers. A portable application can be configured to read its configuration parameters during startup from the same location where its own program files are stored._
[It's Not Sarcasm]

**Reply by vchimishuk** [reply]
I'd suggest you to create your own Transmission statically-linked build, create simple shell-script wrapper which passes "home directory" for Transmission during startup. Looks like this way should work.

**Comment by Pentaphon** [comment]
These guys do portable builds of Transmission and you will see the 4.0 stable build here soon after it is released. 

https://portableapps.com/apps/internet/transmission-portable

**Reply by eooeo** [reply]
I will see a stable 4.0 "Portable" build here. Is it right?

**Reply by Pentaphon** [reply]
That's right. Soon after it goes stable on the official page. They do portable builds for many popular apps.

**Reply by eooeo** [reply]
thank you.
